examples/mistral/mixtral.yml

```diff
@@ -16,12 +16,12 @@ output_dir: ./qlora-out
 
 ## You can optionally freeze the entire model and unfreeze a subset of parameters
 unfrozen_parameters:
-#  - lm_head.*
```

Comment:
The previous example was technically wrong (but worked) because this becomes the following:
`lm_head\.*` and checks if the dot repeats 0+ times